### ISSUE TITLE
What does the `repeat` hyperparam do?

### ISSUE THREAD

**Issue by ethman** [facebookresearch/demucs#207]
## ❓ Questions

Hi Alexandre,

Thanks for this repo; it's a terrific resource for the community! Some of my labmates noticed the `repeat` hyperparameter, used [here](https://github.com/facebookresearch/demucs/blob/main/demucs/train.py#L43) and [set to 2 by default in parser.py](https://github.com/facebookresearch/demucs/blob/main/demucs/parser.py#L95). I didn't find any mention of this in the official paper, but I'm curious what this does exactly. Clearly, each repeat sees the full data in the same order, but other than that, what's the difference between setting `repeat=2` vs just doubling the number of epochs? Is there some advantage to repeating?

Thanks,
Ethan
